### ISSUE TITLE
Sync Error with Alfresco

### ISSUE THREAD

**Issue by Montagnard65** [aegif/CmisSync#740]
Hi there,
I had a new strange error... In the attached file, you can see which error is displayed.

An below, the error log file :
`2017-04-06 18:13:29,270 [10] ERROR CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Could process watcher sync update: C:\XXXXX\07655500 System.IO.FileNotFoundException: Le fichier 'C:\UXXXXX\07655500' est introuvable. Nom de fichier : 'C:\XXXXX\07655500' à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) à System.IO.File.GetAttributes(String path) à CmisSync.Lib.Utils.IsFolder(String path) à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.WatcherSyncUpdate(String remoteFolder, String localFolder, String localPath) 2017-04-06 18:13:29,271 [10] DEBUG CmisSync.StatusIconController [(null)] - StatusIconController OnSyncing 2017-04-06 18:13:29,271 [10] DEBUG CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Processing 'Renamed': C:\XXXXX\07655500 -> C:\XXXXX\YYYYY.xlsm. 2017-04-06 18:13:29,272 [10] INFO CmisSync.Lib.RepoBase [(null)] - Sync Error: CmisSync.Lib.Cmis.BaseException, La référence d'objet n'est pas définie à une instance d'un objet. 2017-04-06 18:13:29,273 [10] ERROR CmisSync.StatusIconController [(null)] - Error syncing 'ZZZZZ': La référence d'objet n'est pas définie à une instance d'un objet. CmisSync.Lib.Cmis.BaseException: La référence d'objet n'est pas définie à une instance d'un objet. ---> System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet. à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.WatchSyncMove(String remoteFolder, String localFolder, String oldPathname, String newPathname) à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.WatcherSync(String remoteFolder, String localFolder) à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.Sync(Boolean syncFull) à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.<>c__DisplayClass6.<.ctor>b__4(Object o, DoWorkEventArgs args) --- Fin de la trace de la pile d'exception interne --- 2017-04-06 18:13:29,276 [10] INFO CmisSync.Lib.RepoBase [(null)] - Partial Sync Complete: C:\XXXXX`
Do you have an idea of what is the problem ?

Thanks.
NB : i'm still using cmissync 2.7.7. with Alfresco 5.2.

**Comment by Montagnard65** [comment]
![erreur](https://cloud.githubusercontent.com/assets/8189200/24956728/455ada08-1f89-11e7-8b1e-53d46715625b.png)

**Comment by nicolas-raoul** [comment]
Sorry I have not seen that error yet I think...
Does it occur with the latest version of CmisSync?
https://bitbucket.org/aegif/cmissync/downloads/

**Comment by Montagnard65** [comment]
No, it doesn't occured with the last version but the 2.7.7. I used this one (and previous versions) because you ask me to track the problem with the sync problem of the file name lenght (#735 issue).

**Comment by cpot** [comment]
We seems to have a similar issue with CMISSync 2.9.4 Alfresco 5.1 Community (GoFAST 3.0)

`2017-04-25 15:49:39,166 [9] ERROR CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Could not crawl sync remote folder: /Sites/_Extranet
System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet.
   à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.ProcessRecoverableException(String logMessage, Exception exception)
   à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlRemoteFolder(IFolder remoteSubFolder, String remotePath, String localFolder, IList1 remoteFolders)
   à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlRemote(IFolder remoteFolder, String remotePath, String localFolder, IList1 remoteFiles, IList1 remoteFolders)
   à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlSync(IFolder remoteFolder, String remotePath, String localFolder)
   à CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlRemoteFolder(IFolder remoteSubFolder, String remotePath, String localFolder, IList1 remoteFolders)
2017-04-25 15:49:39,167 [9] INFO  CmisSync.Lib.RepoBase [(null)] - Sync Error: CmisSync.Lib.Cmis.BaseException, La référence d'objet n'est pas définie à une instance d'un objet.
2017-04-25 15:49:39,167 [9] ERROR CmisSync.StatusIconController [(null)] - Error syncing 'gofast_v3': La référence d'objet n'est pas définie à une instance d'un objet.`

**Comment by cpot** [comment]
Still have the issue
